docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -1,3 +1,69 @@
+= Supported user profile and progressive profiling
+
+The user profile preview feature is promoted to be fully supported and user profile is enabled by default!
+
+In the past months, Keycloak team spent huge amount of effort in polishing user
+profile and making it fully supported now. In this release, we continued the effort started in the last release. Lots of improvements, fixes and
+polishing was done based on the thorought testing and feedback from our awesome community.
+
+Few highlights of the user profile feature:
+* Fine-grained control over the attributes that users and administrators can manage so that you can prevent unexpected attributes, and values, from being set. 
+* Ability to specify what user attributes are managed and should be displayed on the forms to regular users or administrators
+
+* Dynamic forms - Previously, the forms where users created or updated their profiles, contain four basic attributes like username, email, first name and last name. Any additional
+attributes (or removing some of the default attributes) required to create custom theme. Now custom themes may not be needed anymore as users will see exactly the requested attributes based on the requirement
```

Comment:
```suggestion
attributes (or removing some of the default attributes) required to create custom theme. Now custom themes may no longer be needed because users see exactly the requested attributes based on the requirement
```